types/slate/slate-tests.ts

```diff
@@ -114,7 +114,7 @@ const plugin: Plugin = {
     schema: {...schema},
 };
 
-const plugins = [plugin];
+const plugins = [plugin, [plugin, [plugin]]];
```

Comment:
I think we are missing a test case ```plugins = plugin;```, which is why the new commit passed the test.

Reply:
The new commit passed the test, because the fixtures were not invalid per se. I'm by no means a Typescript expert, not to mention DefinitelyTyped, so do you know of a way of testing a supposedly faulty test? 
All I get is the notion to provide code that does not fail neither `tslint` nor `tsc`.

Reply:
Your changes are perfectly fine :) I wrongly thought that originally `plugins` were the type of `Plugin | Plugins[]`. Sorry for the confusion!!

Reply:
At one point they were. 
Thas was part of my ...

> Ugh, I got this wrong, sorry

...initiative. *I* am sorry. And also, sorry for the confusion.